x-pack/plugins/siem/public/graphql/types.ts

```diff
@@ -5405,6 +5405,8 @@ export namespace GetOneTimeline {
 
     pinnedEventsSaveObject: Maybe<PinnedEventsSaveObject[]>;
 
+    status: Maybe<TimelineStatus>;
```

Comment:
@XavierM @andrew-goldstein any ideas why this would have been added? I just moved the `and_or_badge` component out into the `public/common/components` folder and updated references to it. Is it ok to just remove this?

Reply:
yes because now we have a status attributes who have been introduced to eliminate untitled timeline. The status of timeline as right now is `active` | `draft` and we are adding a new one soon who will be `immutable`